packages/checkbox/README.md

```diff
@@ -12,30 +12,36 @@ instead of selecting.
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/checkbox?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/checkbox)
 [![Try it on webcomponents.dev](https://img.shields.io/badge/Try%20it%20on-webcomponents.dev-green?style=for-the-badge)](https://webcomponents.dev/edit/collection/fO75441E1Q5ZlI0e9pgq/jeIGAXHMUrTp6hGMquoD/src/index.ts)
 
-```
+```bash
 yarn add @spectrum-web-components/checkbox
 ```
 
 Import the side effectful registration of `<sp-checkbox>` via:
 
-```
+```ts
```

Comment:
nit: this should be js

Reply:
It's valid syntax for either language and since the rest of the examples use ts, it makes sense to keep it ts here as well for consistency.